metadata.rb

```diff
@@ -1,14 +1,14 @@
-name             'maven'
-maintainer       'Chef Software, Inc.'
+name 'maven'
+maintainer 'Chef Software, Inc.'
 maintainer_email 'cookbooks@chef.io'
-license          'Apache 2.0'
-description      'Installs maven 2 or 3 and includes a maven resource.'
+license 'Apache 2.0'
+description 'Application cookbook which installs and configures Maven.'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '1.3.0'
+version '2.0.0'
 
-depends 'ark',  '~> 0.4'
-depends 'java', '~> 1.13'
-depends 'windows'
+depends 'libarchive', '~> 0.6'
+depends 'poise', '~> 2.2'
```

Comment:
Yeah, I struggled with this a little bit. I agree that it should be but currently the Java cookbook doesn't support installing Java 8 for Ubuntu systems. That's a horrible answer but its the truth. I actually ended up wrapping this in my [java-service cookbook](https://github.com/johnbellone/java-service-cookbook/blob/master/recipes/default.rb). 

I am hoping to get the [Java #286 pull-request approved](https://github.com/agileorbit-cookbooks/java/pull/286) and then I would add that in here.